Client/WriteApiAsync.cs

```diff
@@ -220,7 +220,7 @@ public class WriteApiAsync : IWriteApiAsync
         public Task WriteRecordsAsync(string[] records, WritePrecision precision = WritePrecision.Ns,
             string bucket = null, string org = null, CancellationToken cancellationToken = default)
         {
-            return WriteRecordsAsync(records.ToList(), precision, bucket, org, cancellationToken);
+            return WriteRecordsAsync(records, precision, bucket, org, cancellationToken);
```

Comment:
This causes infinity recursive, please change to:

```suggestion
            return WriteRecordsAsync((IEnumerable<string>)records, precision, bucket, org, cancellationToken);
```